client/my-sites/stats/stats-list/action-link.jsx

```diff
@@ -1,55 +1,32 @@
 /**
  * External dependencies
  */
-import React, { PureComponent, PropTypes } from 'react';
```

Comment:
Yes, this wipes out @timmyc's work, which wasn't my intention :) I hit "use theirs" in Sourcetree but something's gone awry. I'll restore it from master.